tests/validators/literal.test.ts

```diff
@@ -0,0 +1,22 @@
+import { ExpectedValidationError, s } from '../../src';
+
+describe('LiteralValidator', () => {
```

Comment:
Missing test for Date's special case:

https://github.com/sapphiredev/shapeshift/blob/dc7ccaf9f2bac0513fbb58f5ac0676e2fc2b83e2/src/lib/Shapes.ts#L76